content/docs/getting-started/installation/operator/modules/authorizationv2-0.md

```diff
@@ -9,19 +9,66 @@ description: >
 {{% /pageinfo %}}
 ## Install Container Storage Modules Authorization via Container Storage Modules Operator
 
+Storage system credentials can be provided in one of two ways:
+1. Using a SecretProviderClass (for dynamic secrets from external providers)
+2. Using a Kubernetes Secret (for static credentials)
+Only one of the two can be specified at a time.
 
-### Prerequisite
+{{< accordion id="secret-provider-class" title="Using a Secret Provider Class" markdown="true" >}}
```

Comment:
There are a lot of duplicate steps under each accordion. I think it'd better list the installation steps and have this accordion, or tabs, specifically for the step of configuring the storageSystemCredentials. There, an accordion can display the Secret and SecretProviderClass.

In another comment, I asked to use tabs. I think we should be consistent with using tabs or accordions, not both.

Reply:
Fixed